CONTRIBUTING.md

```diff
@@ -292,6 +292,7 @@ checklist.)
 2. Unit tests included and/or updated with changes?
 3. Command line build passes?
 4. Expect to pass code review?
+5. Changes have been smoke-tested?
```

Comment:
In the interest of keeping this checklist short, I'm thinking of keeping the new smoke-testing check and removing "Expect to pass code review?" The latter question just serves as a reminder to think about code style etc, but I don't think it really adds anything to the process to have this (the reviewer will ultimately be responsible for determining if those standards are appropriately adhered-to)